app/Http/Controllers/Airlines/GetAllAirlinesController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers\Airlines;
+
+use App\Models\Airline;
+use App\Transformers\AirlineTransformer;
+use Flugg\Responder\Contracts\Responder;
+use Illuminate\Http\JsonResponse;
+
+class GetAllAirlinesController
+{
+    public function __invoke(Responder $responder): JsonResponse
+    {
+        $airlines = Airline::paginate(8);
+        return $responder->success($airlines, new AirlineTransformer())->respond();
```

Comment:
```suggestion
        $airlines = Airline::paginate(8);

        return $responder
            ->success($airlines, AirlineTransformer::class)
            ->respond();
```
You don't need to pass an instance of a transformer; you can pass it like this, and it'll create the instance for you. This is also how we do it in projects.